Update Azure database for mysql with Azure metrics source compatibility (#5386)
SumoLogic/sumologic-documentation: docs/integrations/microsoft-azure/azure-database-for-mysql.md

Since this source contains logs from multiple regions, ensure that you do not tag this source with the location tag.
:::

### Collecting Error Logs

> Since error logs cannot be emitted to Azure diagnostic logs directly. To access the error logs, navigate to **Server Logs > Error Logs**, download them, then [upload the logs](/docs/send-data/hosted-collectors/http-source/logs-metrics/upload-logs) to a Sumo Logic HTTP source endpoint. Please note that in the HTTP source created to receive the error logs, need to be tagged with the following [fields](#manage-fields) with appropriate values. These values can be copied from appropriate resource for which logs are ingested through Azure Eventhub for log source. : 

Reviewer: ```suggestion 
Error logs cannot be emitted to Azure diagnostic logs directly. To access the error logs, navigate to **Server Logs > Error Logs**, download them, then [upload the logs](/docs/send-data/hosted-collectors/http-source/logs-metrics/upload-logs) to a Sumo Logic HTTP source endpoint. Note that the HTTP source created to receive the error logs needs to be tagged with the following [fields](/docs/manage/fields/#manage-fields) with appropriate values. These values can be copied from the appropriate resource for which logs are ingested through the Azure Event Hub Source for Logs: 
```

<AppInstallNoDataSourceV2/>

As part of the app installation process, the following fields will be created by default:

- `tenant_name`. This field is tagged at the collector level. You can get the tenant name using the instructions [here](https://learn.microsoft.com/en-us/azure/active-directory-b2c/tenant-management-read-tenant-name#get-your-tenant-name).
- `location`. The region to which the resource name belongs to.
- `subscription_id`. ID associated with a subscription where the resource is present.
- `resource_group`. The resource group name where the Azure resource is present.
- `provider_name`. Azure resource provider name (for example, Microsoft.Network).
- `resource_type`. Azure resource type (for example, storage accounts).
- `resource_name`. The name of the resource (for example, storage account name).

> - `service_type`. Type of the service that can be accessed with a Azure resource.

Reviewer: ```suggestion
- `service_type`. Type of the service that can be accessed with an Azure resource.
```
